### PR TITLE
Unwaffle rebase

### PR DESCRIPTION
**Pull request by xiphmont**
The long-awaited patch to add deblocking to the loop filter RDO.  This is good for anywhere from .5% to 1.5% gains without significant additional complexity.

### CODE REVIEW

File: src/tiling/plane_region.rs [hunk 1]
```diff
@@ -308,7 +308,7 @@ macro_rules! plane_region_common {
         self.to_frame_block_offset(TileBlockOffset(BlockOffset { x: 0, y: 0 }))
       }
 
-      #[cfg(feature = "check_asm")]
+      //#[cfg(feature = "check_asm")]
```
Comment (lu-zero) [inline src/tiling/plane_region.rs]: maybe delete it now?

File: src/frame/mod.rs [hunk 2]
```diff
@@ -13,7 +13,7 @@ use crate::context::SB_SIZE;
 use crate::mc::SUBPEL_FILTER_SIZE;
 use crate::util::*;
 
-#[cfg(test)]
+//#[cfg(test)]
```
Comment (lu-zero) [inline src/frame/mod.rs]: Maybe delete it now.

File: src/tiling/plane_region.rs (hunk 1 above)
Comment (xiphmont) [reply src/tiling/plane_region.rs]: done

File: src/frame/mod.rs (hunk 2 above)
Comment (xiphmont) [reply src/frame/mod.rs]: done

**Comment by xiphmont** [comment]
Okay!  Ready for review.

File: src/cdef.rs [hunk 3]
```diff
@@ -333,87 +311,88 @@ pub fn cdef_analyze_superblock<T: Pixel>(
...
-  fi: &FrameInvariants<T>, sb_w: usize, sb_h: usize, tile: &Tile<'_, T>,
-) -> Frame<T> {
-  let sb_h_size =
-    if fi.sequence.use_128x128_superblock { 128 } else { 64 } * sb_w;
-  let sb_v_size =
-    if fi.sequence.use_128x128_superblock { 128 } else { 64 } * sb_h;
-
+// Allocates and returns a new Frame with its own memory that is
+// patterned on the decimation of the Frame backing the passed-in
+// Tile.  The width and hight are in units of 8-pixel (undecimated)
```
Comment (tmatth) [inline src/cdef.rs]: nit: `height`

File: src/cdef.rs [hunk 4]
```diff
@@ -333,87 +311,88 @@ pub fn cdef_analyze_superblock<T: Pixel>(
...
     ],
   }
 }
 
-pub fn cdef_sb_padded_frame_copy<T: Pixel>(
-  fi: &FrameInvariants<T>, sbo: TileSuperBlockOffset, sb_w: usize,
-  sb_h: usize, tile: &Tile<'_, T>, pad: usize,
+// Allocates and returns a new Frame with its own memory that is
+// patterned on the decimation of the Frame backing the passed-in
+// Tile.  The width and hight are in units of 8-pixel (undecimated)
```
Comment (tmatth) [inline src/cdef.rs]: nit: `height`

File: src/cdef.rs (hunk 3 above)
Comment (xiphmont) [reply src/cdef.rs]: fixed

File: src/cdef.rs (hunk 4 above)
Comment (xiphmont) [reply src/cdef.rs]: fixed

**Comment by xiphmont** [comment]
wfflebase-baseline@2020-03-17T01:13:56.753Z -> wfflebase-delayedtest@2020-03-17T04:08:33.366Z

   PSNR | PSNR Cb | PSNR Cr | PSNR HVS |    SSIM | MS SSIM | CIEDE 2000
-0.3529 | -1.7094 | -1.4463 |  -0.4564 | -0.7294 | -0.6658 |    -0.4472

Not *quite* as awesome after rebase (other things caught up?) but still very good.

**Pull request merged**